src/modules.js

```diff
@@ -26,12 +48,12 @@ if (__analytics.debug.isEnabled) {
 
 
 // --- Google Tag Manager ---
-if (__analytics.googleTagManager.isEnabled) {
+if (__analytics.googleTagManager?.isEnabled) {
   const googleAnalyticsModule = function () {
     /* eslint-disable */
     // @formatter:off
     (function(s,o,g,a,m){a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
```

Comment:
R: Please revert the unwanted changes here and in other scripts and also note the `@formatter:off` declaration. Support for this declaration can be enabled in JebBrains IDEs under Preferences > Editor > Code Style > Formatter Control.
You can read more here: https://www.jetbrains.com/help/idea/settings-code-style.html#formatter-control